janus-idp/operator
# Avoid hardcoded images

## Description

Avoid hardcoded images as stated in https://github.com/janus-idp/operator/issues/162

https://issues.redhat.com/browse/RHIDP-1190

## Which issue(s) does this PR fix or relate to

https://github.com/janus-idp/operator/issues/162

## PR acceptance criteria


## How to test changes / Special notes to the reviewer
1. make test
2. deploy operator,  kubectl apply -n backstage -f examples/bs1.yaml, make sure the images used are the same as specified in default-config: quay.io/janus-idp/backstage-showcase:latest and quay.io/fedora/postgresql-15:latest)
